CODIGO/ModLogin.bas

```diff
@@ -309,3 +309,44 @@ Public Sub TransferChar(ByVal Name As String, ByVal DestinationAccunt As String)
     ModAuth.LoginOperation = e_operation.TransferCharacter
     Call connectToLoginServer
 End Sub
+
+Public Sub OnClientDisconnect(ByVal Error As Long)
+    On Error GoTo OnClientDisconnect_Err
+    If (Error = 10061) Then
+        If frmConnect.visible Then
+            Call DisplayError("¡No me pude conectar! Te recomiendo verificar el estado de los servidores en ao20.com.ar y asegurarse de estar conectado a internet.", "connection-failure")
```

Comment:
Can we please localize the strings? we need an english version

Reply:
babel localize this for us